spec/requests/volunteers_spec.rb

```diff
@@ -374,7 +374,7 @@
 
       expect(volunteer.invitation_created_at.present?).to eq(true)
       expect(Devise.mailer.deliveries.count).to eq(1)
-      expect(Devise.mailer.deliveries.first.subject).to eq(I18n.t("devise.mailer.invitation_instructions.subject"))
+      expect(Devise.mailer.deliveries.first.subject).to eq("CASA Console invitation instructions")
```

Comment:
Maybe revert this and `spec/requests/supervisors_spec.rb`, `spec/requests/casa_admins_spec.rb` because devise calls the translation in its default view file which is located in its library files so it's inaccessible.